parlai/tasks/dailydialog/agents.py

```diff
@@ -88,3 +88,43 @@ def share(self):
         shared = super().share()
         shared['data'] = self.data
         return shared
+
+
+class NoSilenceTeacher(Convai2Teacher):
```

Comment:
What do you think about `NoStart`? `NoSilence` is fine but I agree we could have a slightly better name